src/logstash-filters/snippets/doppler.conf

```diff
@@ -0,0 +1,71 @@
+# Parse Cloud Foundry logs from doppler (via https://github.com/SpringerPE/firehose-to-syslog)
+
+json {
+    source => 'syslog_message'
+    add_tag => [ 'cloudfoundry_doppler' ]
+}
+
+if "cloudfoundry_doppler" in [tags] {
```

Comment:
This conditional is mostly unnecessary... tags should always have `cloudfoundry_doppler` since it's coming after the json/add_tag.